run_tests.sh

```diff
@@ -39,7 +39,7 @@ python -m tests.http_server $SERVER_PORT 2> /dev/null &
 echo "Running test server on port $SERVER_PORT"
 
 echo_task "Run unit test suite"
-coverage run -m nose $REPO_ROOT/tests/$1
+coverage run -m nose $REPO_ROOT/tests/$1 --nocapture
```

Comment:
oh snaps, didn't mean to check that in! (It causes print statements to not be swallowed up by nose, so I use it for debugging.)